exercises/roman-numerals/.meta/src/reference/kotlin/RomanNumerals.kt

```diff
@@ -1,4 +1,5 @@
-object RomanNumeral {
+object RomanNumerals {
+
```

Comment:
Colleagues, do we have common project code style settings files for idea/eclipse?

Reply:
I guess we should use [official code style](https://kotlinlang.org/docs/reference/code-style-migration-guide.html#in-gradle).

Reply:
This one does not insert a new line between object/class declaration  and the first method, for example. Not a big deal, actually, when it comes to stubs.

Reply:
Ah, you mean that. In this case it's my personal taste - I think it look better when logical blocks (like function declaration) are not "stuck" to class header/declaration.

But I can see both styles in Kotlin std lib. :)

Reply:
If this empty line does not appear automatically as a result of code formatting - I'm perfectly ok with it. 
But if it is - I'd like to have the same formatter for understandable reasons.